README.md

```diff
@@ -151,6 +171,24 @@ project layout:
 }
 ```
 
+```html
+<!-- index.html -->
+<html>
+  <head>
+    <script type="module">
+      {
+        {
+          headScript;
```

Comment:
This formatting doesn't look right since I assumed the intent was `{{headscript}}`?

Reply:
noooo autolinter!

Reply:
fixed